This patch allows the cpu.stats values to be reset in cgroups
without having to remove and recreate it.  I wrote this patch
because I had a use case where resetting the values was required.
Many other components of cgroups allows stats to be reset, such
as cpuacct.stats.  To reset the stats, simple `echo 0 > cpu.stats`
--- linux-3.12.53.orig/kernel/sched/core.c	2016-01-27 02:56:20.000000000 -0600
+++ linux-3.12.53/kernel/sched/core.c	2016-02-12 16:15:41.630753880 -0600
@@ -7495,6 +7495,21 @@
 
 	return 0;
 }
+
+static int cpu_stats_reset(struct cgroup_subsys_state *css, struct cftype *cft,
+                u64 reset)
+{
+	struct task_group *tg = css_tg(css);
+	struct cfs_bandwidth *cfs_b = &tg->cfs_bandwidth;
+
+	raw_spin_lock(&cfs_b->lock);
+	cfs_b->nr_periods = 0;
+	cfs_b->nr_throttled =0;
+	cfs_b->throttled_time = 0;
+	raw_spin_unlock(&cfs_b->lock);
+
+	return 0;
+}
 #endif /* CONFIG_CFS_BANDWIDTH */
 #endif /* CONFIG_FAIR_GROUP_SCHED */
 
@@ -7546,6 +7561,7 @@
 	{
 		.name = "stat",
 		.read_map = cpu_stats_show,
+		.write_u64 = cpu_stats_reset,
 	},
 #endif
 #ifdef CONFIG_RT_GROUP_SCHED
